### PR TITLE
Feature/cab 3746

### PR DESCRIPTION
**Pull request by huiaic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +31,46 @@ For example, to run the same test with 10 concurrent threads:
...
+```
+execution.iterations=1000 # number of rows from id file to run [default: 20]
+scenarios.delete.think-time=1000ms # time delay between each call [default: 100ms]
+scenarios.delete.data-sources.path=/path/id.csv # path to id file [default: ./data/id.csv]
+scenarios.delete.data-sources.loop=true # loop over in case of end-of-file reached if true, stop executing if false [default: false]
+```
+
+#### Search
+
+1. Create the search body payload file under `content-api-crud/payload/search-body.json`
```
Comment (agagne) [inline README.md:55]: Will you create an example search?

File: content-api-crud/search.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+execution:
+- concurrency: 1
```
Comment (agagne) [inline content-api-crud/search.yml:2]: Would it be possible to configure concurrency via the command line, or do you need to update it in the yml?

File: README.md [hunk 3]
```diff
@@ -11,9 +11,10 @@ Install [Taurus](https://gettaurus.org/docs/Installation/)
 
 ### content-api-crud
 
+`run-config.yml` runs a all CRUD actions on content-api.
```
Comment (agagne) [inline README.md]: What is bzt? Do we need to install some special software? Would you please add that to the readme?

Comment (huiaic) [reply README.md]: yes, setup instructions are in the existing readme

File: content-api-crud/search.yml (hunk 2 above)
Comment (huiaic) [reply content-api-crud/search.yml:2]: yes, that is documented in the existing readme.

**Review by agagne — APPROVED** [review]

**Pull request merged**